crates/cheatnet/src/forking/cache.rs

```diff
@@ -25,13 +26,13 @@ struct ForkCacheContent {
     class_hash_at: HashMap<String, String>,
     compiled_contract_class: HashMap<String, String>,
     compiled_class_hash: HashMap<String, String>,
-    block_info: Option<BlockInfo>,
+    block_info: Option<ForgeBlockInfo>,
 }
 
 impl ForkCacheContent {
     fn new() -> Self {
         Self {
-            cache_version: "1.0".to_string(),
+            cache_version: "2.0".to_string(),
```

Comment:
The structure of cache changed, so we want to have a new version so that the old ones are not used causing parsing errors